doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md

```diff
@@ -5,8 +5,17 @@ PART 1: GITHUB UI
 {% endhint %}
 
 ## Learning videos
+### Default branch and pull request base branch
 
-Set 2 learning videos are not available yet. Check back on or before the 12th of October 2022.
+{% embed url="https://www.youtube.com/watch?v=hRFBG3lMq9k" %}
+YouTube Video
+{% endembed %}
+
+### Workflow common practices and best practices
+
+{% embed url="https://www.youtube.com/watch?v=Ej396Vra1oQ" %}
+YouTube Video
+{% endembed %}
```

Comment:
Neatly done! @oyenuga17 
One more thing before I approve, let the pull request title start with `Fix: `

Reply:
Alright

Reply:
I just did it..